client/my-sites/checkout/src/components/test/secondary-cart-promotions.tsx

```diff
@@ -20,8 +23,32 @@ jest.mock( '@automattic/calypso-config', () => {
 	return mock;
 } );
 
+jest.mock( '@automattic/data-stores', () => {
+	return {
+		...jest.requireActual( '@automattic/data-stores' ),
+		Plans: {
+			...jest.requireActual( '@automattic/data-stores' ).Plans,
+			usePricingMetaForGridPlans: jest.fn(),
+		},
+	};
+} );
+
 describe( 'SecondaryCartPromotions', () => {
 	const store = applyMiddleware( thunk )( createStore )( storeData );
+	const queryClient = new QueryClient();
+	const Wrapper = ( { children } ) => (
+		<QueryClientProvider client={ queryClient }>{ children }</QueryClientProvider>
+	);
+
+	beforeEach( () => {
+		jest.clearAllMocks();
+		Plans.usePricingMetaForGridPlans.mockImplementation( () => ( {
+			[ PLAN_PERSONAL ]: {
+				originalPrice: { full: 24000, monthly: 2000 },
+				discountedPrice: { full: 12000, monthly: 1000 },
+			},
+		} ) );
```

Comment:
@sirbrillig previous comment outdated already. 😬 
Some updates in the last commit -> now using and mocking only the pricing-meta hook.